backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxUtils.java

```diff
@@ -195,15 +230,22 @@ private static void setText(XWPFRun run, String text) {
         }
     }
 
-    private static void addFormattedText(XWPFRun run, String text, String fontFamily, int fontSize, boolean bold) {
+    private static void addFormattedText(XWPFRun run, String text, String fontFamily, int fontSize, boolean bold, String rrggbbColor) {
         run.setFontSize(fontSize);
         run.setFontFamily(fontFamily);
         run.setBold(bold);
+        if (rrggbbColor != null) {
+            run.setColor(rrggbbColor);
+        }
         setText(run, text);
     }
 
+    private static void addFormattedText(XWPFRun run, String text, int fontSize, boolean bold, String rrggbbColor) {
```

Comment:
the parameter `fontSize` is always `FONT_SIZE`.

Reply:
I only overloaded the existing method. Didn't want to do any major work on this class. I guess this was designed to accept non-default font sizes, but never actually used.